packages/uniorg-parse/src/parser.ts

```diff
@@ -1520,7 +1589,12 @@ class Parser {
     }
     const value = getOrgEntity(m.groups!.value1 ?? m.groups!.value2);
     if (!value) return null;
-    return u('entity', { useBrackets: hasBrackets, ...value });
+    const end = begin + m[0].length;
```

Comment:
❗🐛 does not take into account backoff when `!hasBrackets`.
```suggestion
    const end = this.r.offset();
```